sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/PartitionKeyBuilder.java

```diff
@@ -0,0 +1,90 @@
+package com.azure.cosmos.models;
+
+import com.azure.cosmos.implementation.Undefined;
+import com.azure.cosmos.implementation.routing.PartitionKeyInternal;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PartitionKeyBuilder {
```

Comment:
For all public contract please add documentation.

Reply:
Also missing:
@Beta(value = Beta.SinceVersion.V4_12_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)

Reply:
Added

Reply:
Added the public documentation..